requirements.txt

```
@@ -27,7 +27,7 @@ lightkube-models >= 1.22.0.4
 # Code: https://github.com/canonical/operator/
 # Docs: https://ops.rtfd.io/
 # Deps: charm
-ops < 2.5.0  # https://github.com/canonical/ops-scenario/issues/48
+ops >= 2.6.0  # https://github.com/canonical/ops-scenario/issues/48
```

Now that https://github.com/canonical/ops-scenario/issues/48 is fixed we can drop the comment?
```suggestion
ops >= 2.6.0
```